### PR TITLE
Db chapter

### PR DESCRIPTION
**Pull request by cookingkode**

### CODE REVIEW

File: db/cassandra.dml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+
+CREATE KEYSPACE roster WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 1};
+
+create table employees (
```
Comment (petomalina) [inline db/cassandra.dml]: CREATE TABLE should be upper case

File: db/cassandra.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gocql/gocql"
+)
+
+// Employee struct
+type User struct {
```
Comment (petomalina) [inline db/cassandra.go]: All comments of public structures should start with their name, e.g. `User is a ...`

File: db/gorm_associations.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	_ "fmt"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/mysql"
+)
+
+type HotelChain struct {
```
Comment (petomalina) [inline db/gorm_associations.go]: Missing comments

**Review by petomalina — CHANGES_REQUESTED** [review]
It would be useful to add more comments to lines of code that access the database. E.g. `db.AutoMigrate(&User{})` and its comment `Migrate the Schema` may be right, but its really weak definition of whats really going on under the hood (using reflection to get fields, map to db types, ...).

I would recommend adding more comments to these spots

File: db/cassandra.dml (hunk 1 above)
Comment (cookingkode) [reply db/cassandra.dml]: done

File: db/cassandra.go (hunk 2 above)
Comment (cookingkode) [reply db/cassandra.go]: done

**Pull request merged**